README.md

```diff
@@ -20,13 +22,14 @@ optional arguments:
   -s, --southbound      Use this if you're going southbound
   -w WALK_TIME, --walk-time WALK_TIME
                         The walk time (in minutes) from where you are to the
-                        departure station
+                        departure station.
+			If you omit this argument, your walk-time will be estimated based on your current location.
```

Comment:
This should be in line with the line above. I just copy-pasted this from the command `./dart.py --help`, which makes it more consistent